src/Parser.php

```diff
@@ -1218,7 +1234,7 @@ function ($parentNode) {
                 return null;
```

Comment:
Previously it would return null if `$node->nameParts` was null, ie if `parseDelimitedList` returned null. I think this check should be for `$nameParts` now? Otherwise I agree with not using $node->nameParts to hold a temporary value.

Reply:
Yes, it should. Missed that.